bower.json

```diff
@@ -47,8 +47,6 @@
     "patternfly-timeline": "~1.0.3",
     "phantomjs-polyfill": "~0.0.2",
     "qs": "~0.3.10",
-    "rx-angular": "rx.angular#~1.1.3",
```

Comment:
Angular RX was in first iteration of ui-components, we had to watch on scope for some changes and it was better to use Rx's observable, instead Angular's (they use setTimeout or something strange in $watch function). But later we did not used it, so feel free to remove it, we'll remove it in ui-components as well.